1.8/test/import-without-vendor-app/s2i-build

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+s2i build -e IMPORT_URL='github.com/cpuguy83/go-md2man' ${s2i_args} file://./app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
```

Comment:
It would be great to test also incremental builds.

Reply:
If you won't object I would create later separate pull request.

Reply:
No problem with this. Thanks in advance.